GLOSSARY.md

```diff
@@ -124,3 +124,13 @@ Swift is the so-called object storage that can be used by Nova. This is responsi
 ## Ceph OSD
 
 A Ceph OSD (Object Storage Daemon) logically represents a storage device in a Ceph cluster, which can logically be a hard disk, which is the ideal case. In other cases it can also be a raid, which however leads to considerable performance limitations due to caching or other raid optimization.
+
+## Personas
+
+|Persona|Description|
+|SCS Operator|The SCS Operator is the one who owns and operates a standardized cloud environment.|
+|SCS Consumer|The SCS Consumer consumes a standardized SCS environment and operates and orchestrates applications on top of it. The SCS Consumer is typically a customer or user of the SCS Operator.|
+|SCS Integrator|The SCS Integrator assists in or is building up a standardized cloud environment. The SCS Integrator can be 2nd or 3rd level support for the SCS Operator.|
+|SCS Project|The SCS Project oversees the overall activities around the Sovereign Cloud Stack.|
+|SCS Developer|The SCS Developer actively contributes to technical elements of the Sovereign Cloud Stack.|
```

Comment:
What kind of piped formatting is this? I have corrected some sentences and added a list for better readability.

```suggestion
|Persona|Description|

- |SCS Operator|The SCS Operator owns and operates a standardized cloud environment.|
- |SCS Consumer|The SCS Consumer consumes a standardized SCS environment, operates and orchestrates applications on top of it. The SCS Consumer is typically a customer or user of the SCS Operator.|
- |SCS Integrator|The SCS Integrator assists in or is building up a standardized cloud environment. The SCS Integrator can be 2nd or 3rd level support for the SCS Operator.|
- |SCS Project|The SCS Project oversees the overall activities around the Sovereign Cloud Stack.|
- |SCS Developer|The SCS Developer actively contributes to technical elements of the Sovereign Cloud Stack.|
```

Reply:
sorry, i botchered the table. fix with the next commit.